package.json

```diff
@@ -17,15 +17,18 @@
   },
   "dependencies": {
     "brace": "^0.8.0",
+    "classnames": "^2.2.5",
```

Comment:
I don't know really well how NPM js dependencies works, but I think this problem can be solved by adding classnames and flexboxgrid in 'dependencies' instead only in 'devDependencies'. right?